### PR TITLE
Pass welcome details to onConnect() and onReconnectSuccess(). close #73

### LINKED ISSUE #73: Welcome details on connect
Can we have "welcome details" as an argument to `onConnect()` and `onReconnectSuccess()`? It's useful to get some session details once client connects. My current workaround to this is calling `wamp.session.get`, but I would like to avoid odd call, when all needed info is in "welcome".

If you accept PR for this, I could make one.

Thanks!

### PR DESCRIPTION
**Pull request by azamat-sharapov**

### CODE REVIEW

File: test/wampy-json-test.js [hunk 1]
```diff
@@ -32,13 +32,46 @@ describe('Wampy.js [with JSON serializer]', function () {
...
+                onConnect: function (welcomeDetails) {
+                    expect(welcomeDetails).to.be.an('object');
+                    done();
+                },
+                ws: WebSocketModule.WebSocket,
+                serializer: new JsonSerializer()
+            })
+        });
+
+        it('passes welcome details to onReconnectSuccess() callback', function (done) {
```
Comment (azamat-sharapov) [inline test/wampy-json-test.js]: @KSDaemon for some reason, this test isn't passing. Can you check, please?

Comment (KSDaemon) [reply test/wampy-json-test.js]: Yep, i'll take a look

**Comment by KSDaemon** [comment]
Despite the falling test, thank you for your work! I'll merge it and fix tests. See next commit, in case you are interested what was the problem.

**Comment by azamat-sharapov** [comment]
@KSDaemon thanks! I think you should write small instruction on how to write tests for the project, because it took me a _while_ to realize, that I needed to populate "send-data" file, otherwise simply adding another simple test was breaking all other tests. It would help future contributors.

**Comment by KSDaemon** [comment]
Yes! You're right! Right now tests are not a really self-contained unit-tests, because i use one shared websocket mock... I'll try to rethink it all on accasion...

**Pull request merged**